cmd/tsdbrelay/doc.go

```diff
@@ -22,15 +22,16 @@ Usage:
 
 The flags are:
 	-b="bosun"
-		Target Bosun server. Can specify port with host:port.
+		Target Bosun server. Can specify as host, host:port, or https://host:port.
 	-t=""
-		Target OpenTSDB server. Can specify port with host:port.
+		Target OpenTSDB server. Can specify as host, host:port or https://host:port.
 	-l=":4242"
 		Listen address.
 	-v=false
 	    Enable verbose logging
 	-r=""
 		Additional relays to send data to, comma seperated. Intended for secondary data center replication. Only response from primary tsdb server wil be relayed to clients.
+		Examples: hostA:port,https://hostB:port,hostC#data-only,https://hostD:8080#metadata-only
```

Comment:
This is currently driven by command line flags. It is a bit of a hack, but one that only takes a few lines and still looks pretty clean. Moving to a full config file is another option, but would be a bigger refactor and not be backwards compatible.